production/helm/loki/values.yaml

```diff
@@ -206,6 +209,9 @@ loki:
     max_cache_freshness_per_query: 10m
     split_queries_by_interval: 15m
 
+  # -- Provides a reloadable runtime configuration file for some specific configuration
+  runtimeConfig: {}
```

Comment:
I agree with this decision to make this `loki.runtimeConfig` rather than top level :+1: